### PR TITLE
fix: remove 'Field' from column type names

### PR DESCRIPTION
**Pull request by samuelhwilliams**
This strips out 'Field' from the descriptive names and changes a few that don't seem entirely accurate... Open to being challenged on any of the renames. Should all of the 'char'-y fields just be described as 'Text'?

### CODE REVIEW

File: explorer/schema.py [hunk 1]
```diff
@@ -74,23 +74,23 @@ def schema_info(connection_alias, schema=None, table=None):
...
-    BIGINT: 'IntegerField',
-    NUMERIC: 'FloatField',
-    DOUBLE_PRECISION: 'FloatField',
-    FLOAT: 'FloatField',
-    BOOLEAN: 'BooleanField',
-    DATE: 'DateField',
-    TIMESTAMP: 'TimestampField',
-    Geometry: 'GeometryField',
+    ENUM: 'Enum',
+    CHAR: 'Char',
```
Comment (abbas123456) [inline explorer/schema.py:78]: Char probably isn't the friendliest word to use, I think Text is better

File: explorer/schema.py [hunk 2]
```diff
@@ -74,23 +74,23 @@ def schema_info(connection_alias, schema=None, table=None):
...
+    VARCHAR: 'Char',
+    String: 'Text',
+    UUID: 'UUID',
+    TEXT: 'Text',
+    ARRAY: 'Array',
+    INTEGER: 'Integer',
+    SMALLINT: 'Integer',
+    BIGINT: 'Integer',
+    NUMERIC: 'Float',
```
Comment (abbas123456) [inline explorer/schema.py:87]: Perhaps decimal is a more common term?

Comment (samuelhwilliams) [reply explorer/schema.py:87]: Yeah I wondered the same - let's go with Decimal.

File: explorer/schema.py (hunk 1 above)
Comment (samuelhwilliams) [reply explorer/schema.py:78]: Yep, sounds good.

**Pull request merged**